src/test/java/seedu/fma/logic/commands/CommandTestUtil.java

```diff
@@ -30,10 +30,10 @@
 
 public class CommandTestUtil {
     public static final String VALID_EXERCISE_SIT_UP = "Sit ups";
-    public static final String VALID_EXERCISE_CRUNCHES = "Crunches";
+    public static final String VALID_EXERCISE_FLYING_KICKS = "Jumping jacks";
```

Comment:
Refactor to VALID_EXERCISE_JUMPING... for clarity?

Reply:
Yeap for sure, thanks for checking!